exts/propagation.py

```diff
@@ -81,6 +82,16 @@ async def solarweather(self, ctx: commands.Context):
         embed.set_image(url=self.n0nbh_sun_url)
         await ctx.send(embed=embed)
 
+    @commands.command(name="drapmap", aliases=["drap", "radioblackout", "blackout", "ionosphere"],
```

Comment:
```suggestion
    @commands.command(name="drapmap", aliases=["drap", "radioblackout", "blackout"],
```
That alias is too vague. I'm on the fence about `radioblackout` and `blackout`, not sure if they should be removed or not.
If they go, the continuation on the next line should be moved back to this one.